modules/tinymce/src/themes/silver/main/ts/ui/general/Notification.ts

```diff
@@ -30,7 +30,7 @@ export interface NotificationSketchSpec extends Sketcher.SingleSketchSpec {
 // tslint:disable-next-line:no-empty-interface
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +22]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This line should change as well:
```ts
  readonly level: 'info' | 'warn' | 'warning' | 'error' | 'success';
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/tinymce/tinymce/10061?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
It's defined as defaultedStringEnum, so it should be optional